internal/controllers/v2controllers/chart_release_test.go

```diff
@@ -537,69 +537,55 @@ func (suite *chartReleaseControllerSuite) TestChartReleaseGetOtherValidSelectors
-	//	db.Truncate(suite.T(), suite.db)
-	//	suite.seedClusters(suite.T())
-	//	suite.seedEnvironments(suite.T())
-	//	suite.seedCharts(suite.T())
-	//	suite.seedChartReleases(suite.T())
-	//
-	//	before, err := suite.ChartReleaseController.Get(datarepoDevChartRelease.Name)
-	//	assert.NoError(suite.T(), err)
-	//	assert.Equal(suite.T(), "latest", *before.TargetChartVersionUse)
-	//	edited, err := suite.ChartReleaseController.Edit(datarepoDevChartRelease.Name, EditableChartRelease{
-	//		TargetChartVersionUse: testutils.PointerTo("exact"), TargetChartVersionExact: testutils.PointerTo("11.22.33"),
-	//	}, auth.GenerateUser(suite.T(), false))
-	//	assert.NoError(suite.T(), err)
-	//	assert.Equal(suite.T(), "exact", *edited.TargetChartVersionUse)
-	//	assert.Equal(suite.T(), "11.22.33", *edited.TargetChartVersionExact)
-	//	after, err := suite.ChartReleaseController.Get(datarepoDevChartRelease.Name)
-	//	assert.NoError(suite.T(), err)
-	//	assert.Equal(suite.T(), "exact", *after.TargetChartVersionUse)
-	//	assert.Equal(suite.T(), "11.22.33", *after.TargetChartVersionExact)
-	//})
-	//suite.Run("edit to suitable chart release", func() {
-	//	db.Truncate(suite.T(), suite.db)
-	//	suite.seedClusters(suite.T())
-	//	suite.seedEnvironments(suite.T())
-	//	suite.seedCharts(suite.T())
-	//	suite.seedChartReleases(suite.T())
-	//
-	//	suite.Run("unsuccessfully if not suitable", func() {
-	//		before, err := suite.ChartReleaseController.Get(datarepoProdChartRelease.Name)
-	//		assert.NoError(suite.T(), err)
-	//		assert.Nil(suite.T(), before.TargetAppVersionCommit)
-	//		_, err = suite.ChartReleaseController.Edit(datarepoProdChartRelease.Name, EditableChartRelease{
-	//			TargetAppVersionCommit: testutils.PointerTo("abc"),
-	//		}, auth.GenerateUser(suite.T(), false))
-	//		assert.ErrorContains(suite.T(), err, errors.Forbidden)
-	//		notEdited, err := suite.ChartReleaseController.Get(datarepoProdChartRelease.Name)
-	//		assert.NoError(suite.T(), err)
-	//		assert.Nil(suite.T(), notEdited.TargetAppVersionCommit)
-	//	})
-	//	suite.Run("successfully if suitable", func() {
-	//		edited, err := suite.ChartReleaseController.Edit(datarepoProdChartRelease.Name, EditableChartRelease{
-	//			TargetAppVersionCommit: testutils.PointerTo("abc"),
-	//		}, auth.GenerateUser(suite.T(), true))
-	//		assert.NoError(suite.T(), err)
-	//		assert.Equal(suite.T(), "abc", *edited.TargetAppVersionCommit)
-	//	})
-	//})
-	//suite.Run("unsuccessfully if invalid", func() {
-	//	db.Truncate(suite.T(), suite.db)
-	//	suite.seedClusters(suite.T())
-	//	suite.seedEnvironments(suite.T())
-	//	suite.seedCharts(suite.T())
-	//	suite.seedChartReleases(suite.T())
-	//
-	//	_, err := suite.ChartReleaseController.Edit(datarepoProdChartRelease.Name, EditableChartRelease{
-	//		TargetAppVersionUse: testutils.PointerTo("something obviously incorrect"),
-	//	}, auth.GenerateUser(suite.T(), true))
-	//	assert.ErrorContains(suite.T(), err, errors.BadRequest)
-	//})
+	suite.Run("successfully", func() {
```

Comment:
This test had been commented from when ChartRelease didn't actually have any editable fields. It does not, so I re-enabled it and tweaked it to work with the current fields. It doesn't impact or test the new functionality here at all.